synapse/rest/media/v1/preview_url_resource.py

```diff
@@ -315,9 +316,12 @@ def render_OPTIONS(self, request):
 
         with self.media_storage.store_into_file(file_info) as (f, fname, finish):
             try:
-                logger.debug("Trying to get url '%s'", url)
+                logger.debug("Trying to get preview for url '%s'", url)
                 length, headers, uri, code = await self.client.get_file(
-                    url, output_stream=f, max_size=self.max_spider_size
+                    url,
+                    output_stream=f,
+                    max_size=self.max_spider_size,
+                    headers={"Accept Language": self.url_preview_accept_language},
```

Comment:
Should this be "Accept-Language", not "Accept Language"?

Reply:
Hah, yes I think so :)